docs/deploy.md

```diff
@@ -146,14 +183,18 @@ For the official homeworld servers:
  * Log into the server directly with your SSH keys
    - Verify the host keys based on the text printed before the login console
 
+For example, ``ssh root@egg-sandwich.mit.edu``. You might need to remove previous ECSDA keys from known_hosts if you've set up the cluster before.
+
 ## Setting up the supervisor node
 
 Set up the keysystem:
 
     $ spire seq keysystem
+      # if it asks you for your SSH key password at every step, you likely already have ssh-agent running; if so, you can do "ssh-add" to add your identity to the agent so that doesn't happen.
```

Comment:
Huh? It doesn't necessarily follow that asking for your password at every step means you _do_ have an agent running. Perhaps also talk about agents before talking about this?